lib/charms/operator_libs_linux/v2/snap.py

```diff
@@ -409,6 +409,25 @@ def unhold(self) -> None:
         """Remove the refresh hold of a snap."""
         self._snap("refresh", ["--unhold"])
 
+    def alias(self, application: str, alias: Optional[str] = None) -> None:
```

Comment:
AFAIK, the library now has been improved. No need to call subprocess. CC: @dragomirp  , right?